src/source/PeerConnection/Rtcp.c

```diff
@@ -250,21 +249,18 @@ STATUS onRtcpTwccPacket(PRtcpPacket pRtcpPacket, PKvsPeerConnection pKvsPeerConn
                         recvOffset += 2;
                         break;
                     case TWCC_STATUS_SYMBOL_NOTRECEIVED:
-                        DLOGV("statusVector packetSeqNum %u not received", packetSeqNum);
-                        if (pKvsPeerConnection->onPacketNotReceived != NULL) {
-                            pKvsPeerConnection->onPacketNotReceived(pKvsPeerConnection->onPacketNotReceivedCustomData, packetSeqNum);
-                        }
+                        DLOGV("statusVector packetSeqNum %u not received %lu", packetSeqNum, referenceTime);
+                        twcc->twccPacketBySeqNum[packetSeqNum].remoteTimeKvs = TWCC_PACKET_LOST_TIME;
+                        twcc->lastReportedSeqNum = packetSeqNum;
                         break;
                     default:
                         DLOGD("statusVector unhandled statusSymbol %u", statusSymbol);
                 }
                 if (recvDelta != MIN_INT16) {
-                    DLOGV("statusVector packetSeqNum %u recvDelta %d usec", packetSeqNum,
-                          KVS_CONVERT_TIMESCALE(recvDelta, TWCC_TICKS_PER_SECOND, MICROSECONDS_PER_SECOND));
-                    if (pKvsPeerConnection->onPacketReceived != NULL) {
-                        pKvsPeerConnection->onPacketReceived(pKvsPeerConnection->onPacketReceivedCustomData, packetSeqNum,
-                                                             KVS_CONVERT_TIMESCALE(recvDelta, TWCC_TICKS_PER_SECOND, MICROSECONDS_PER_SECOND));
-                    }
+                    referenceTime += KVS_CONVERT_TIMESCALE(recvDelta, TWCC_TICKS_PER_SECOND, HUNDREDS_OF_NANOS_IN_A_SECOND);
+                    DLOGV("statusVector packetSeqNum %u received %lu", packetSeqNum, referenceTime);
```

Comment:
Can we change this to `DLOGS`? Will this generate a spew of logs?

Reply:
I used `DLOGV` to debug and finetune the algorithm in real-world conditions.
i will set to `DLOGS` for now but bear with me, this will keep cropping up